docs/.vuepress/enhanceApp.js

```diff
@@ -0,0 +1,42 @@
+const { applyToWindow, instanceRegister } = require('./handsontable-manager');
+
+applyToWindow();
+
+const buildRegisterCleaner = register => (to, from) => {
+  if (to.path === from.path) {
+    return;
+  }
+  if (register === undefined) {
+    // eslint-disable-next-line
+    console.warn('The register doesn\'t exists');
+
+    return;
+  }
+  register.destroyAll();
+};
+
+const buildActiveHeaderLinkHandler = () => {
+  let activeLink = '';
+
+  return (to, from) => {
+    if (to.hash !== from.hash) {
+      if (activeLink) {
+        activeLink.classList.remove('active');
+      }
+      // eslint-disable-next-line
+      activeLink = document.querySelector(`.table-of-contents a[href="${to.hash}"]`);
+
+      if (activeLink) {
+        activeLink.classList.add('active');
+      }
```

Comment:
Shouldn't this be handled by active link plugin? It's some kind of another workaround?

Reply:
The active link plugin does not work for us, and this is a fixation. 


I guess it works only for a common Vuepress table of content solution (in the left sidebar).